pvactools/lib/netmhc_pan_version.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+
+class NetMHCPanVersion:
```

Comment:
You'll want to rename this class `NetMHCIIPanVersion` to avoid confusion with the `NetMHCpan` algorithm and rename classes/variables/arguments in other places as well to match.